src/style.css

```diff
@@ -0,0 +1,118 @@
+.page {
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+  --button-color: #00acdc;
+  --main-color: #060b35;
+  --secondary-color: #616070;
+}
+
+body {
+  margin: 0;
+}
+
+.section {
+  align-items: center;
+  width: 200px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.section--block {
+  box-sizing: border-box;
+  padding: 16px;
+}
+
+.product__image {
+  display: block;
+  width: 100%;
+  padding-top: 16px;
+  height: 134px;
+}
+
+.product__item {
+  margin: 0;
+  padding-top: 40px;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+}
+
+.product__code {
+  color: var(--secondary-color);
+  margin: 4px 0 0;
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.product__rating {
+  display: flex;
+  justify-content: space-between;
+  font-size: 10px;
+  align-items: center;
+  padding: 16px 0 24px;
+}
+
+.stars {
+  margin: 0;
+  display: flex;
+  height: 16px;
+  width: 96px;
+}
+
+.stars__star {
+  margin: 2px;
+  width: 16px;
+  height: 16px;
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+}
+
+.stars--4 > :nth-child(-n+4) {
+  background-image: url(images/star-active.svg);
+}
+
+.product__review {
+  color: var(--main-color);
+}
+
+.cost {
+  display: flex;
+  justify-content: space-between;
+  padding-bottom: 16px;
+}
+
+.cost__price {
+  color: var(--secondary-color);
+  font-size: 12px;
+  line-height: 18px;
+}
+
+.cost__number {
+  color: var(--main-color);
+  font-size: 16px;
+  font-weight: 700;
+}
+
+.tobuy__button {
```

Comment:
You really don't need to create separate block for just one button. Better to make it element of `product` block.